tests/problem1Tests.rb

```diff
@@ -0,0 +1,50 @@
+require 'test/unit'
+require '../src/problem1'
+
+class Problem1Tests < Test::Unit::TestCase
+
+  def setup
+    @problem1 = Problem1.new
+    @testDataInteger1 = [1,4,7,4,10,6]
+    @testDataInteger2 = (1..100).to_a
+    @testDataNotInteger1 = %w(1 2 3 4)
+    @testDataNotInteger2 = [1,nil,3,4,5,6]
+  end
+
+  def teardown
+
+  end
+
+  def test_hasExpectedSumResult1
```

Comment:
In a good test name at a minimum you should have:

- specify the state (context) of the test
- specify the expected behavior